src/model/core/discharge/discharge.jl

```diff
@@ -31,7 +31,7 @@ function discharge!(EP::Model, inputs::Dict, setup::Dict)
     # Variable costs of "generation" for resource "y" during hour "t" = variable O&M
     @expression(EP,
         eCVar_out[y = 1:G, t = 1:T],
-        (inputs["omega"][t]*(var_om_cost_per_mwh(gen[y]) * vP[y, t])))
+        inputs["omega"][t] * var_om_cost_per_mwh(gen[y]) * vP[y, t])
```

Comment:
Something here. But the brackets weren't needed? In this case it's better to do `(a * b) * c` than `a * (b * c)`?